src/files2move2bin/Config.TRACS

```diff
@@ -77,14 +77,14 @@
 Depth = 300.0   # Double in micrometers
 
 # Width of the read-out strips
-Width = 25.0   # Double in micrometers
+Width = 500 #25.0   # Double in micrometers
```

Comment:
No worries, it's nice having somebody go through the code and learn more about GitHub at the same time. :D